### DISCUSSION TITLE
Pinia devtool display of date objects

### DISCUSSION THREAD

**Discussion by Esgator** [vuejs/pinia#1810] [Help and Questions]
Hallo 
I had a confussing interaction with the vue devtool where i was in the process of changing my store from vuex to pinia and having important date objects which where diffrently displayed in the pinia store then in the vuex. 

In vuex it's displayed as a object where i can see all the elements. (see below)
![vuexStore](https://user-images.githubusercontent.com/78217443/202462508-8dbf152d-ef5c-47f8-be5d-9cda93774abb.png)

In pinia i only see the String from the "$d" element. (see below)
![piniaStore](https://user-images.githubusercontent.com/78217443/202462834-0ce28689-9b59-4bca-8afe-2a348ac9a3d2.png)

Is that intentional? Because it was quite confussing to me. Anyone else had that experence? Or is it just me. Complitly fine by the way if thats the case :)
Maybe displaying the Object with all it's elements might be, at least in my humble obinion, a good idea.

Have a great day everybody.

**Comment by posva** [comment]
Where is that `Date` coming from? all this `$D`, `$H`, etc seem to be non native.
